homeassistant/components/bosch_shc/strings.json

```diff
@@ -10,7 +10,7 @@
       },
       "credentials": {
         "data": {
-          "password": "Password of the Smart Home Controller"
+          "password": "[%key:common::config_flow::data::password%]"
```

Comment:
Could this be here to solve a possible confusion with an another password?

Reply:
Added back as `data_description`.

Reply:
Right, but is there a possible confusion or not?

Reply:
Could be, I searched for this and it seems there is a password you need to connect devices from apps and another one to login to the device which confused some others (e.g. https://community.bosch-smarthome.com/t5/funktionen-und-nutzung/systempasswort-zurück-setzen/td-p/13701 (German))